### ISSUE TITLE
Running with docker

### ISSUE THREAD

**Issue by kakka0903** [search-rug/search-rug.github.io#1]
Running the `jekyll-debug.sh` script on an M1 macbook greeted me with the following warning:

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

And, after installing ruby gems the process exited with:

```bash
[...]
Bundle complete! 2 Gemfile dependencies, 90 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
ruby 3.1.1p18 (2022-02-18 revision 53f5fc4236) [x86_64-linux-musl]
jekyll 3.9.3 | Error:  Whoops, we can't understand your command.
jekyll 3.9.3 | Error:  invalid option: --disable-disk-cache
jekyll 3.9.3 | Error:  Run your command again with the --help switch to see available options.
/usr/gem/gems/mercenary-0.3.6/lib/mercenary/program.rb:31:in `go': invalid option: --disable-disk-cache (OptionParser::InvalidOption)
	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
	from /usr/gem/gems/jekyll-3.9.3/exe/jekyll:15:in `<top (required)>'
	from /usr/local/bundle/bin/jekyll:27:in `load'
	from /usr/local/bundle/bin/jekyll:27:in `<main>'
```

Running the project on a Ubuntu machine with an amd64 architecture eliminated the warning, but gave the same error. Running the without docker works fine:

```bash
rvm install "ruby-2.7.2"
rvm use 2.7
bundle install 
gem install jekyll
jekyll serve
```

**Comment by kakka0903** [comment]
@feitosa-daniel do you have ideas for what is going wrong here? How should we move forward to fixing it?

**Comment by feitosa-daniel** [comment]
Fixed with 5eb60e2c6c304a1a27a9e4086de880ebd7952a41
